.idea/copyright/Apache_2_0_ORT_Server.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (C) &amp;#36;today.year The ORT Server Authors (See &lt;https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE&gt;)&#10;&#10;Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);&#10;you may not use this file except in compliance with the License.&#10;You may obtain a copy of the License at&#10;&#10;    https://www.apache.org/licenses/LICENSE-2.0&#10;&#10;Unless required by applicable law or agreed to in writing, software&#10;distributed under the License is distributed on an &quot;AS IS&quot; BASIS,&#10;WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.&#10;See the License for the specific language governing permissions and&#10;limitations under the License.&#10;&#10;SPDX-License-Identifier: Apache-2.0&#10;License-Filename: LICENSE" />
+    <option name="myName" value="Apache-2.0-ORT-Server" />
+  </copyright>
+</component>
```

Comment:
The missing empty line is intentional to keep the format exactly as IntelliJ is generating this file.